api/src/test/java/uk/gov/dhsc/htbhf/hmrc/testhelper/TestConstants.java

```diff
@@ -5,11 +5,27 @@
 
 public class TestConstants {
 
+    public static final String SIMPSONS_SURNAME = "Simpson";
+    public static  final String SIMPSONS_ADDRESS_LINE_1 = "742 Evergreen Terrace";
+    public static  final String SIMPSONS_ADDRESS_LINE_2 = "Mystery Spot";
+    public static  final String SIMPSONS_TOWN_OR_CITY = "Springfield";
+    public static final String SIMPSONS_POSTCODE = "AA11AA";
+
     public static final LocalDate LISA_DOB = LocalDate.parse("1985-12-31");
     public static final String LISA_NINO = "EB123456C";
     public static final String LISA_FORENAME = "Lisa";
-    public static final String LISA_SURNAME = "Simpson";
+    public static final String LISA_SURNAME = SIMPSONS_SURNAME;
```

Comment:
I'm not sure this really helps, can we not just use `SIMPSONS_SURNAME`?

Reply:
Removed in `LISA_SURNAME` in favour of `SIMPSONS_SURNAME`